src/util/color/colorpalette.cpp

```diff
@@ -0,0 +1,22 @@
+#include "colorpalette.h"
+
+constexpr mixxx::RgbColor kColorMixxxRed = mixxx::RgbColor(0xC50A08);
```

Comment:
I added "Mixxx" in case we later want to add other predefined pallettes (e.g. the Serato or Rekordbox palette).

Reply:
@uklotzde Is keeping the "Mixxx" prefix okay or should I remove it? It shows that this color belongs to the Mixxx palette.

Reply:
The constants still should be moved into an anonymous namespace since they are not accessible as symbols from outside this module.

Optionally this anonymous namespace might by enclosed in an outer `mixxx` namespace. Then repeating "Mixxx" in all names becomes. But it doesn't really matter as long as they symbols are not exposed.

Reply:
The diff is outdated. See https://github.com/mixxxdj/mixxx/commit/b8ee8d656a2591f212411cac29da1ad6faf33ad9